index.html

```diff
@@ -0,0 +1,178 @@
+                    </ul>
+                </div>
+                <div class="container desktop-nav-container">
+
+                    <a href="/index.html"><img
+                            src="https://99designs-blog.imgix.net/blog/wp-content/uploads/2020/05/attachment_62260073-e1588694361186.png?auto=format&q=60&fit=max&w=930"
+                            class="nav-logo"></a>
+
+
+
+                    <ul class="desktop-menu-items">
+                        <!-- <li><a href="#">Home</a></li> -->
+                        <li><a class="active" href="/about.html">About</a></li>
+                        <li><a href="#">Program</a></li>
+                        <li><a href="#">Sponsor</a></li>
+                        <li><a href="#">News</a></li>
+                        <li><a href="#">CC Campaign</a></li>
+                    </ul>
+                </div>
+
+            </div>
+        </nav>
+    </header>
+    <section class="headline">
+        <div class="container">
+            <div class="headline-welcome">
+                <h2>"Hello! Digital Marketers"</h2>
+                <h1 class="title">DIGITAL MARKETING <br> GLOBAL SUMMIT <br> 2023</h1>
+            </div>
+            <div class="headline-text">
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione, minima voluptatum ab tenetur quasi
+                    porro?
+                    Deleniti consequatur ex quam tenetur dolor corporis reprehenderit molestiae veritatis, molestias
+                    excepturi
+                    repellat illo illum.</p>
+            </div>
+            <div class="headline-address">
+                <h3>2023.10.15(THU) ~ 16(FRI)</h3>
+                <p>@ Science museum of Ethiopia</p>
+            </div>
+
+        </div>
+    </section>
+    <section class="main">
+        <div class="container">
+
+
+            <h2>Main Program</h2>
+            <hr>
+            <div class="cards">
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        podium
+                    </span>
+                    <h3>Lecture</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        gallery_thumbnail
+                    </span>
+                    <h3>CC Exhibition</h3>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. architecto assumenda accusamus mollitia
+                        eveniet
+                        est ullam ipsa.</p>
+                </div>
+                <div class="card">
+                    <span class="material-symbols-outlined">
+                        forum
+                    </span>
```

Comment:
- [ ] Kindly take a second look at the mobile version of the description, you need to align it so that they will have equal width.
## Your design
![image](https://user-images.githubusercontent.com/86200356/233104503-636a9e75-6db2-4d81-807b-695ee78c0db0.png)


## Behance design

![image](https://user-images.githubusercontent.com/86200356/233104246-27b87c85-3e6f-4536-9818-4f366857a65e.png)